tests/test_chromatic.py

```diff
@@ -275,15 +276,15 @@ def test_chromatic_add():
                                                zenith_angle=zenith_angle)
 
     # create final profile
-    final = galsim.Convolve([bdgal, chromatic_PSF])
+    final = galsim.Convolve(bdgal, chromatic_PSF)
     image = galsim.ImageD(stamp_size, stamp_size, scale=pixel_scale)
     image = final.drawImage(bandpass, image=image)
 
     do_pickle(bulge)
     do_pickle(disk)
-    do_pickle(bdgal)
+    # do_pickle(bdgal)
     do_pickle(chromatic_PSF)
-    do_pickle(final)
+    # do_pickle(final)
```

Comment:
Oops.. Did we lose even more picklability for chromatic objects?  I guess there are more use of lambda functions now?  This is going to be a problem when we want to use chromatic objects in a multiprocessing framework.  

Any chance we can recover picklability by deferring the application of lambdas until we actually need them?  Or is it too baked in?